libdevcore/RLP.h

```diff
@@ -277,9 +277,9 @@ class RLP
 		return fromBigEndian<_T>(p);
 	}
 
-	int64_t toPositiveInt64(int _flags = Strict) const
-	{
-		int64_t i = toInt<int64_t>(_flags);
+    int64_t toUint64(int _flags = Strict) const
```

Comment:
Don't change this name please, it was ok and it doesn't return uint64.

Changing the name was ok in the testeth code where you changed the implementation...